### PR TITLE
DBZ-6250: Missing a keyword which can be used as column name

### PR DESCRIPTION
**Pull request by sondn98**
See [DBZ-6250](https://issues.redhat.com/browse/DBZ-6250)

### CODE REVIEW

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4 [hunk 1]
```diff
@@ -2783,7 +2783,7 @@ keywordsCanBeId
     | THAN | TP_CONNECTION_ADMIN | TRADITIONAL | TRANSACTION | TRANSACTIONAL | TRIGGERS | TRUNCATE | UNBOUNDED | UNDEFINED | UNDOFILE
     | UNDO_BUFFER_SIZE | UNINSTALL | UNKNOWN | UNTIL | UPGRADE | USA | USER | USE_FRM | USER_RESOURCES | VALIDATION
     | VALUE | VAR_POP | VAR_SAMP | VARIABLES | VARIANCE | VERSION_TOKEN_ADMIN | VIEW | VIRTUAL | WAIT | WARNINGS
-    | WITHOUT | WORK | WRAPPER | X509 | XA | XA_RECOVER_ADMIN | XML
+    | WITHOUT | WORK | WRAPPER | X509 | XA | XA_RECOVER_ADMIN | XML | GEOMETRY
```
Comment (Naros) [inline debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4:2786]: Can you please place the new `GEOMETRY` keyword alphabetically in the rule set  like all others?

**Review by Naros — CHANGES_REQUESTED** [review]
Hi @sondn98 I found one change I'd like to see fixed, otherwise it LGTM.

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4 (hunk 1 above)
Comment (sondn98) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4]: It's okay, I've changed it upon your comment

**Comment by mfvitale** [comment]
@Naros should this be added also to the [upstream](https://github.com/antlr/grammars-v4)?

**Comment by jpechane** [comment]
@sondn98 Applied, thanks! Could you please also create an upstream PR to https://github.com/antlr/grammars-v4 and add reference to it here?

**Pull request merged**